File: LibEasyGlide/src/main/java/me/bzcoder/easyglide/progress/OnProgressListener.java
package me.bzcoder.easyglide.progress;

/**
 * 
 * @author : BaoZhou
 * @date : 2019/6/4 9:25
 */
public interface OnProgressListener {
    void onProgress(boolean isComplete, int percentage, long bytesRead, long totalBytes);
}

File: LibEasyGlide/src/main/java/me/bzcoder/easyglide/progress/OnProgressListener.kt
package me.bzcoder.easyglide.progress

/**
 *
 * @author : BaoZhou
 * @date : 2019/6/4 9:25
 */
interface OnProgressListener {
    fun onProgress(isComplete: Boolean, percentage: Int, bytesRead: Long, totalBytes: Long)
}